runtime/doc/lua.txt

```diff
@@ -3013,6 +3006,35 @@ vim.fs.parents({start})                                     *vim.fs.parents()*
         (`nil`)
         (`string?`)
 
+vim.fs.root({source}, {marker})                                *vim.fs.root()*
+    Find the first parent directory containing a specific "marker", relative
+    to a buffer's directory.
+
+    Example: >lua
+        -- Find the root of a Python project, starting from file 'main.py'
+        vim.fs.root(vim.env.PWD .. 'main.py', {'pyproject.toml', 'setup.py' })
```

Comment:
is this misleading, i.e. should we be hinting at `getcwd()` insted of $PWD? also seems good to mention joinpath()

```suggestion
        vim.fs.root(vim.fs.joinpath(vim.env.PWD, 'main.py'), {'pyproject.toml', 'setup.py' })
```

Reply:
ffc16b6b5ddb5f1ee59566b2069db953462f06dc